strawberryfields/apps/graph/utils.py

```diff
@@ -151,13 +151,24 @@ def is_subgraph(subgraph: Iterable, graph: nx.Graph):
 
 
 def is_clique(graph: nx.Graph) -> bool:
-    """Determines if the input graph is a clique. A clique of n nodes has :math:`n*(n-1)/2` edges.
+    """Determines if the input graph is a clique. A clique of :math:`n` nodes has
+     :math:`n*(n-1)/2` edges.
+
+    Example usage:
+
+    .. code-block::
+
+        >>> from strawberryfields.apps.graph import utils
```

Comment:
This import signature is a bit odd, why not just 
```suggestion
        >>> from strawberryfields.apps.graph.utils import is_clique
```

Reply:
Actually, ideally you wouldn't have this import in code examples at all, since it's kind of a given. Do you need it for doctests to work?

Reply:
I think it's best to make import statements explicit: this will be useful for less experienced users that can use these examples to build their own code.

In that case, isn't
`from strawberryfields.apps.graph import utils` 
preferable if it's part of code that calls other functions in `utils`? <-- Genuine question, not rhetorical

Reply:
Maybe, but it's important to consider the user's namespace. What would happen if every library recommended importing a library called `utils`? In this case since it's just a single example function, I think it's better to import just the function you're calling.

Reply:
Ok, following also @josh146 's advice, I'll remove import statements

Reply:
Have removed! Thanks!